### DISCUSSION TITLE
Fedora Official Repos Should be Added

### DISCUSSION THREAD

**Discussion by Tahinli** [zed-industries/zed#25635] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

Hi, right now Zed Fedora packages are in Terra which is unofficial Fedora repository. Fedora users shouldn't need to install extra repo for Zed. Even some hobby projects are in official repo. As you know official Fedora repos have many users not only in Fedora itself but commercial distros also. I hope we will going to see Zed in official repo. I'm not sure if there is useful for this project but here is some resource about how to do:

https://jrfernandez.com/packaging-rust-program-for-fedora

### If applicable, add mockups / screenshots to help present your vision of the feature

If a great software targets a platform, it **must** support the platform **correctly**.

**Comment by notpeter** [comment]
This is something on our radar, but not something we have yet decided to prioritize.

**Comment by Jacalz** [comment]
Sorry to comment on an old issue but I just want to chime in and say that adding this would be greatly appreciated!
